src/app/modules/shared-components/components/small-section/small-section.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="progress-section-container">
+  <div class="progress-section-header" [ngClass]="{disabled: disabled, 'cursor-pointer': collapsed}" (click)="onCollapse()">
```

Comment:
Beware, a lot of class names contain `progress-section` but the component is not `progress-section` anymore

Reply:
fixed